changelogs/fragments/70180-collection-list-more-robust.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ansible-galaxy - do not require mandatory keys in ``galaxy.yml`` for a collection (https://github.com/ansible/ansible/issues/70180)
```

Comment:
any collection?

Reply:
whoops

Reply:
Changelog now mentions the important parts - applies to source collections when listing collections.